src/main/java/org/opentripplanner/standalone/config/RouterConfig.java

```diff
@@ -65,6 +70,10 @@ public double streetRoutingTimeoutSeconds() {
         return streetRoutingTimeoutSeconds;
     }
 
+    public URI getBikeRentalServiceDirectoryUrl() {
```

Comment:
Add a short javadoc to this method including that it configure a Sandbox feature.